collectors/auth0/auth0_collector.js

```diff
@@ -37,8 +39,9 @@ class Auth0Collector extends PawsCollector {
 
     pawsGetLogs(state, callback) {
         let collector = this;
+        const hostname = process.env.paws_endpoint.toLowerCase().replace(HOSTNAME_REGEXP, '');
```

Comment:
Hmm... might break if auth0 is case sensitive on endpoints; perhaps better to add i to the regex replace?